src/main/java/com/codeborne/selenide/conditions/CssClass.java

```diff
@@ -0,0 +1,34 @@
+package com.codeborne.selenide.conditions;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Driver;
+import org.openqa.selenium.WebElement;
+
+public class CssClass extends Condition {
+  private final String expectedCssClass;
+
+  public CssClass(String expectedCssClass) {
+    super("css class");
+    this.expectedCssClass = expectedCssClass;
+  }
+
+  @Override
+  public boolean apply(Driver driver, WebElement element) {
+    String actualCssClasses = element.getAttribute("class");
+    return actualCssClasses != null && contains(actualCssClasses.split(" "), expectedCssClass);
+  }
+
+  @Override
+  public String toString() {
+    return getName() + " '" + expectedCssClass + '\'';
+  }
+
+  private <T> boolean contains(T[] objects, T object) {
```

Comment:
this generic should better go into a `util` or `helper` class

Reply:
or use 

[`ArrayUtils.contains(Object[] array, Object objectToFind)`](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/ArrayUtils.html#contains-java.lang.Object:A-java.lang.Object-)

Reply:
@muescha Thank you for your comments! I will try to make these improvements in a next pull request.

Reply:
@muescha Finally, I have reviewed your comments. 
Initially Selenide didn't depend on commons-lang.jar. I didn't want to introduce a new dependency just because of 2 tiny methods. That's why I wrote those 2 methods by myself (`ArrayUtils.contains` and  `StringUtils.defaultString`).

Now Selenide depends on commons-lang.jar, but not directly (through BMP and HtmlUnit). Most of Selenide users automatically get commons-lang.jar, but there can be some users who exclude BMP and HtmlUnit, and they would also exclude commons-lang.jar.

I am not sure if this is a strong argument to avoid using commons-lang.jar directly in Selenide code?...